mgmt/RecordsConfig.cc

```diff
@@ -1257,6 +1257,11 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_NODE, "proxy.node.version.manager.build_person", RECD_STRING, nullptr, RECU_NULL, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
+  //# manager: backoff configuration.
+  {RECT_CONFIG, "proxy.node.config.manager_exponential_sleep_ceiling", RECD_INT, "60", RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
+  ,
+  {RECT_CONFIG, "proxy.node.config.manager_retry_cap", RECD_INT, "5", RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
```

Comment:
Given a retry cap of 5 what is the maximum about of time it is going to keep on retrying (from the time of the first crash, to the last retry)?

Reply:
From the first crash, it will retry exponentially till reaches the ceiling time, in this case `60`, the retry cap is applied only when it reaches the max ceiling time.
With this configuration it will be like this:
```
[Mar  2 23:54:22.367] traffic_manager NOTE: Relaunching proxy after 1.434 sec...
[Mar  2 23:54:27.188] traffic_manager NOTE: Relaunching proxy after 2.543 sec...
[Mar  2 23:54:33.309] traffic_manager NOTE: Relaunching proxy after 4.509 sec...
[Mar  2 23:54:41.343] traffic_manager NOTE: Relaunching proxy after 8.176 sec...
[Mar  2 23:54:52.918] traffic_manager NOTE: Relaunching proxy after 16.180 sec...
[Mar  2 23:55:12.216] traffic_manager NOTE: Relaunching proxy after 32.870 sec...
[Mar  2 23:55:48.213] traffic_manager NOTE: Relaunching proxy after 60.627 sec...
[Mar  2 23:56:52.168] traffic_manager NOTE: Relaunching proxy after 60.168 sec...
[Mar  2 23:57:55.600] traffic_manager NOTE: Relaunching proxy after 60.250 sec...
[Mar  2 23:58:59.210] traffic_manager NOTE: Relaunching proxy after 60.825 sec...
[Mar  3 00:00:03.222] traffic_manager NOTE: Relaunching proxy after 60.276 sec...
```
_variance is also being show in this example_

Reply:
So roughly 363 seconds based on the defaults.

Reply:
Correct.